packages/commonwealth/client/scripts/views/Sublayout.tsx

```diff
@@ -194,6 +196,17 @@ const Sublayout = ({ children, isInsideCommunity }: SublayoutProps) => {
             )}
             {children}
           </div>
+          <CWGrowlTemplate
+            headerText="Launch Contests On Farcaster!"
+            bodyText="You can now host contests directly on Farcaster to reach and engage your followers.
+            They can submit entries,
+            vote for their favorites, and earn rewards, all without leaving the page."
+            buttonText="Enter $MOCHI Contest"
+            buttonLink="https://www.google.com/"
```

Comment:
is this link correct?

Reply:
@masvelio My bad, should have mentioned that above. The link is just a stand in for now since the actual Mochi contest isn't made yet. On monday I'll be out so I just made this growl ticket quickly, and Malik is going to update the link and get it hotfixed monday.